ui/lib/pki/addon/routes/configuration/edit.js

```diff
@@ -4,5 +4,21 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+// import { withConfirmLeave } from 'core/decorators/confirm-leave';
 
-export default class PkiConfigurationEditRoute extends Route {}
+// @withConfirmLeave()
```

Comment:
`withConfirmLeave` is commented out here. Is that intentional?

Reply:
Nope! Thank you for the reminder, when I finished developing I want to comment it back in!